app/controllers/applications_controller.rb

```diff
@@ -7,6 +7,13 @@ def create
     @application = Application.new(params.require(:application).permit(:name,
       :email, :language_de, :language_en, :attended_before, :rejected_before, :level,
       :comments, :os, :needs_computer))
-    @application.save!
+
+    if @application.save
+      redirect_to application_path
```

Comment:
We did this a bit differently, instead of redirecting on success, we just did nothing so the default would happen: rendering `create.html.erb`, with the idea of writing a success message there (which you did). This is also 🆗 (but then you don't need the `create.html.erb` file).